data-prepper-plugins/otel-metrics-source/src/main/java/com/amazon/dataprepper/plugins/source/otelmetrics/OTelMetricSourceConfig.java

```diff
@@ -0,0 +1,200 @@
+/*
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  The OpenSearch Contributors require contributions made to
+ *  this file be licensed under the Apache-2.0 license or a
+ *  compatible open source license.
+ *
+ *  Modifications Copyright OpenSearch Contributors. See
+ *  GitHub history for details.
+ */
+
+package com.amazon.dataprepper.plugins.source.otelmetrics;
+
+import com.amazon.dataprepper.model.configuration.PluginSetting;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class OTelMetricSourceConfig {
+    static final String REQUEST_TIMEOUT = "request_timeout";
+    static final String PORT = "port";
+    static final String SSL = "ssl";
+    static final String USE_ACM_CERT_FOR_SSL = "useAcmCertForSSL";
+    static final String ACM_CERT_ISSUE_TIME_OUT_MILLIS = "acmCertIssueTimeOutMillis";
+    static final String HEALTH_CHECK_SERVICE = "health_check_service";
+    static final String PROTO_REFLECTION_SERVICE = "proto_reflection_service";
+    static final String SSL_KEY_CERT_FILE = "sslKeyCertChainFile";
+    static final String SSL_KEY_FILE = "sslKeyFile";
+    static final String ACM_CERT_ARN = "acmCertificateArn";
+    static final String ACM_PRIVATE_KEY_PASSWORD = "acmPrivateKeyPassword";
+    static final String AWS_REGION = "awsRegion";
+    static final String THREAD_COUNT = "thread_count";
+    static final String MAX_CONNECTION_COUNT = "max_connection_count";
+    static final String ENABLE_UNFRAMED_REQUESTS = "unframed_requests";
+    static final int DEFAULT_REQUEST_TIMEOUT_MS = 10000;
+    static final int DEFAULT_PORT = 21890;
```

Comment:
This is the default port that the Trace source uses. I think these should be default values. Otherwise, if you have two pipelines - one for trace and metrics - Data Prepper will fail to start with defaults.

Perhaps `21891` would be a good port number.

Reply:
Done